__tests__/app.test.js

```diff
@@ -0,0 +1,40 @@
+const request = require('supertest');
+const app = require('../app/app');
+
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+const db = require('../db/connection');
+
+afterAll(() => db.end());
+
+beforeEach(() => seed(testData));
+
+describe('GET/api/topics', () => {
+  test('status 200, returns an array of object in correct format', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((result) => {
+        const topics = result.body;
+        expect(topics).toBeInstanceOf(Array);
+        topics.forEach((topic) => {
+          expect(topic).toEqual(
+            expect.objectContaining({
+              slug: expect.any(String),
+              description: expect.any(String),
+            })
```

Comment:
if you're looping through an array in a test you need to make sure it's not empty or you'll get a false positive, could you add that to this test please